api/api/tests/vuln_tests.py

```diff
@@ -31,34 +32,41 @@ def tearDown(self) -> None:
         self.manager.delete()
         self.other_user.delete()
 
-    def test_create_valid_vulnerability(self):
```

Comment:
is it not working ? why? :D

Reply:
i can lookup a fix if you need help

Reply:
Yes, I'm getting "unknow vuln_type" and I don't know why. Check it if you can